### DISCUSSION TITLE
Markdown function supporting inline HTML?

### DISCUSSION THREAD

**Discussion by VincentEthier** [datapane/datapane#45] [Help] [answered]
Hi,

I'm new to Datapane and while trying to insert markdowns in my report, it looked like all the inline HTML in my .md file weren't interpreted as HTML... Is there a way to get around this issue?

Here's the code snippet :

```python
import datapane as dp

from SimpsonThirdGraph import fig1

parag = (open("OpMed.md","r"))
paragString = parag.read()

report = dp.Report(
    dp.Markdown(paragString),
    dp.Plot(fig1),
)

report.save(path='test2.html', open=False)

```

**Comment by lanthias** [comment]
Hi there, thanks for the suggestion! 

We actually have a few components which we've built and are being released in the next couple of days:
- An HTML component which you can pass raw HTML into (unlike the Markdown renderer, which will sanitise some HTML)
- A BigNumber component for presenting KPIs and numbers
- Pandas HTML tables

The first one should do exactly what you need, so we will let you know when it has been released.

**Comment by lanthias** [comment, marked as answer]
Hi there, we just launched our new HTML component which should handle your use-case. You read about it in our 0.9.0 release notes here: https://blog.datapane.com/announcing-datapane-0-9/
 
Let me know if I can help with anything!

**Reply by VincentEthier** [reply]
Can’t wait to try it. Thanks for your work!
